themes/finna2/templates/Helpers/organisation-info.phtml

```diff
@@ -21,9 +21,9 @@
     <div class="<?=$this->showDetails ? 'info-col ' : ''?>info">
       <div class="organisation-dropdown-holder">
         <div class="organisation dropdown">
-            <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><input type="hidden"><span></span><i class="fa fa-arrow-down"></i></a>
-            <ul class="dropdown-menu" role="menu">
-            </ul>
+          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><input type="hidden"><span class="sr-only"><?=$this->transEsc('organisation_info_selected_service_point')?></span><span></span><i class="fa fa-arrow-down"></i></a>
+          <ul class="dropdown-menu" role="menu" aria-label="<?=$this->transEsc('organisation_info_select_service_point')?>">
```

Comment:
Sori, en huomannut tätä aiemmin: attribuuteille tehtävä `$this->escapeHtmlAttr($this->translate('...'))`

Reply:
jeps, muutettu